### PR TITLE
Make all telemetry tests support multiple clusters

### PR DESCRIPTION
**Pull request by zirain**
**Please provide a description of this PR:**

Make all telemetry tests support multiple clusters

### CODE REVIEW

**Comment by zirain** [comment]
note: this might be the root case
```
2022-02-26T01:12:33.815382Z	info	tf	istioctl ([x workload entry configure -f /logs/artifacts/telemetry-stackdriver-vm-97e0ad/_suite_context/server-vm-config-906351300/workloadgroup.yaml -o /logs/artifacts/telemetry-stackdriver-vm-97e0ad/_suite_context/server-vm-config-906351300/v1-1970620067]): completed after 0.0097s
2022-02-26T01:12:33.815454Z	info	tf	istioctl error: Error: failed to automatically determine the --clusterID: fetch injection template: configmaps "istio-sidecar-injector" not found
```

**Comment by zirain** [comment]
```console
2022-02-26T06:27:56.683014Z	warn	Could not auto-detect IP for. Use --ingressIP to manually specify the Gateway address to reach istiod from the VM.%!(EXTRA string=istiod.istio-system.svc, string=istio-system)
Configuration generation into directory /logs/artifacts/telemetry-stackdriver-vm-604dc7/_suite_context/server-vm-config-3582892505/v1-2880498147 was successful
2022-02-26T06:27:56.683148Z	info	tf	istioctl ([x workload entry configure -f /logs/artifacts/telemetry-stackdriver-vm-604dc7/_suite_context/server-vm-config-3582892505/workloadgroup.yaml -o /logs/artifacts/telemetry-stackdriver-vm-604dc7/_suite_context/server-vm-config-3582892505/v1-2880498147 --clusterID config]): completed after 0.0179s
2022-02-26T06:28:03.024164Z	warn	failed deploying echo WLE for istio-echo-2-57183/server to pimary cluster: failed applying YAML to cluster external: error when creating "/logs/artifacts/telemetry-stackdriver-vm-604dc7/_suite_context/WorkloadEntry.2559474714.yaml": namespaces "istio-echo-2-57183" not found: 
2022-02-26T06:28:03.113257Z	info	tf	=== DONE: Deploy echo instances ===
```

File: tests/integration/telemetry/stackdriver/vm/main_test.go [hunk 1]
```diff
@@ -121,7 +121,6 @@ func TestMain(m *testing.M) {
 		// https://github.com/istio/istio/issues/35923. Since IPv6 has no external connectivity, we are "not on GCP"
 		// in the sense that we cannot access the metadata server
 		Label(label.IPv4).
-		RequireSingleCluster().
 		RequireMultiPrimary().
```
[the remark this answers is not in the document]
Comment (douglas-reid) [reply tests/integration/telemetry/stackdriver/vm/main_test.go:124]: Perhaps?  Let's open a separate issue for tracking that and assign it to me.

**Review by douglas-reid — COMMENTED** [review]
Thanks for tackling this!

File: tests/integration/telemetry/stackdriver/vm/main_test.go (hunk 1 above)
[the remark this answers is not in the document]
Comment (zirain) [reply tests/integration/telemetry/stackdriver/vm/main_test.go:124]: linked to https://github.com/istio/istio/issues/37602

[the remark this answers is not in the document]
Comment (douglas-reid) [reply tests/integration/telemetry/stackdriver/vm/main_test.go:124]: LG. Let's leave this as `RequireSingleCluster` and then get this PR merged.

**Comment by zirain** [comment]
@istio/wg-test-and-release-maintainers @istio/wg-security-maintainers can we get this in? leave the rest part to @douglas-reid

**Review by howardjohn — APPROVED** [review]

**Review by bianpengyuan — APPROVED** [review]

**Pull request merged**